cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -90,6 +90,8 @@
   private final byte[] nameAsTxChangePrefix;
   // Whether read-less increments should be used when increment() is called
   private final boolean enableReadlessIncrements;
+  //the current snapshot version
+  protected final Long snapshotVersion;
```

Comment:
Why it is in buffering table? Why does the buffering table need to be aware of which snapshot version to use?